packages/vue/src/DataDisplay/Table/OcTable.vue

```diff
@@ -114,9 +114,9 @@ onMounted(() => onScroll());
 <template>
   <div
     ref="scrollTable"
-    class="grid text-oc-text flex-col border border-oc-gray-200 isolate z-10"
+    class="flex text-oc-text flex-col border border-oc-gray-200 isolate z-10"
     :class="[
-      isSticky ? 'overflow-x-auto' : 'overflow-hidden',
+      isSticky ? 'overflow-x-auto grid' : 'overflow-hidden',
```

Comment:
It should break the current table layout?

Reply:
No

Reply:
I use grid because we have this issue when isSticky = true

![image](https://github.com/hit-pay/orchid/assets/16191053/30ffd969-a2d7-4d8e-bd57-aa1863368b51)

Reply:
if `isSticky` is not passed, the layout is changed to `flex`, so there are not backward compatible

Reply:
cc @kewcoder

Reply:
This bug is due to the fact that the content of the rows differs in width from the headers, and we can set classes for each header (static or relative width so that the width of the TableCell is the same)

Reply:
@mykhailo-diakovych In this table all header columns had static width